### PR TITLE
Saved View left panel fixes

### PR DESCRIPTION
**Pull request by berggren**
This PR changes how saved searches are handled in the left menu.
* No route is pushed when clicked. Using setView() instead.
* Added 3-dots menu with an option to link to the saved search

Screenshot 2023-04-26 at 09 52 07

### CODE REVIEW

File: timesketch/frontend-ng/src/components/LeftPanel/SavedSearches.vue [hunk 1]
```diff
@@ -30,16 +30,36 @@ limitations under the License.
...
+              ><div class="mt-1">{{ savedSearch.name }}</div></v-col
+            >
+            <v-col cols="1">
+              <v-menu offset-y>
+                <template v-slot:activator="{ on, attrs }">
+                  <v-btn small icon v-bind="attrs" v-on="on">
+                    <v-icon small>mdi-dots-vertical</v-icon>
+                  </v-btn>
+                </template>
+                <v-card>
```
Comment (jkppr) [inline timesketch/frontend-ng/src/components/LeftPanel/SavedSearches.vue:51]: What do you need the v-card component here for? This submenu should work without as well.

File: timesketch/frontend-ng/src/components/LeftPanel/SavedSearches.vue [hunk 2]
```diff
@@ -30,16 +30,36 @@ limitations under the License.
...
+                  <v-list dense class="mx-auto">
+                    <v-list-item style="cursor: pointer" @click="copySavedSearchUrlToClipboard(savedSearch.id)">
+                      <v-list-item-title>
+                        <v-icon>mdi-link-variant</v-icon> Copy link to this search
```
Comment (jkppr) [inline timesketch/frontend-ng/src/components/LeftPanel/SavedSearches.vue:55]: With other submenus in the Left panel (e.g. Sigma rule) we use small icons.

```suggestion
                        <v-icon small>mdi-link-variant</v-icon> Copy link to this search
```

Also, the sigma rules make use of the `v-list-item-icon` component. Which results in some space between icon and title. I'm fine with both, but we should decide on one general way for submenus.
Option A: Have icons in the `v-list-item-icon` components and separated from the `v-list-item-title` component.
Option B: Have icons within the `v-list-item-title` component.

**Review by jkppr — CHANGES_REQUESTED** [review]
Generally looks great. Some small nits below.

File: timesketch/frontend-ng/src/components/LeftPanel/SavedSearches.vue (hunk 1 above)
Comment (berggren) [reply timesketch/frontend-ng/src/components/LeftPanel/SavedSearches.vue:51]: fixed

File: timesketch/frontend-ng/src/components/LeftPanel/SavedSearches.vue (hunk 2 above)
Comment (berggren) [reply timesketch/frontend-ng/src/components/LeftPanel/SavedSearches.vue]: Fixed, changed to be more consistent.

**Review by jkppr — APPROVED** [review]
lgtm

**Pull request merged**